requirement.txt

```diff
@@ -0,0 +1,37 @@
+appdirs==1.4.4
```

Comment:
A good idea is to put packages here at the time of installation. That way you will only be mentioning packages which are absolutely necessary and not the dependent ones. pip automatically fetches the dependent one. When you do pip freeze > requirements.txt, it puts all the packages here which becomes a lil confusing to manage.